bibtex-actions.el

```diff
@@ -276,16 +294,23 @@ key associated with each one."
                             (list pdf note link))"	") suffix))))
 
 (defvar bibtex-actions--candidates-cache nil
```

Comment:
```suggestion
(defvar bibtex-actions--candidates-cache 'uninitialized
```

Reply:
Why this?

Reply:
For clarity and also because it makes the refresh `bibtex-actions--get-candidates` and `bibtex-actions--refresh` somewhat simpler since we lambda's

Reply:
Gotcha; I just batch-committed these.